Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -1257,6 +1275,7 @@ AmbiguityFiltersDict: !dictionary
   entries:
     '^\d{4}$': '(\d\.\d{4}|\d{4}\.\d)'
     '\b(lunch)$': '(?<!\b(op|om|voor|ana(ar)?|rond)\b\s)(lunch)'
+    '^(morgen|middag|avond|nacht|dag)\b': '\b(goeden?\s*(morgen|middag|avond|nacht|dag))\b'
```

Comment:
Good catch! Small improvement could be to add 'goeiemorgen' / 'goeiemiddag', used in Flemish (nl-be). But I can add this in a follow up PR as well.

https://www.vlaanderen.be/taaladvies/goedemorgen-goeiemorgen-goede-morgen-goeie-morgen-goedenmorgen
https://www.vlaanderen.be/taaladvies/goedemiddag-goeiemiddag-goede-middag-goeie-middag-goedenmiddag

Reply:
added